components/fxa-client/android/src/main/java/mozilla/appservices/fxaclient/FirefoxAccount.kt

```diff
@@ -176,14 +176,25 @@ class FirefoxAccount(handle: FxaHandle, persistCallback: PersistCallback?) : Aut
     /**
      * Fetches the token server endpoint, for authentication using the SAML bearer flow.
```

Comment:
I understand this was already here, but wait..what..SAML? 😕

Reply:
(Sorry, just to be clear, I'm not asking you to do anything about this, I'm just noting my surprise and confusion)